lollipop/types.py

```diff
@@ -184,7 +184,7 @@ def load(self, data, *args, **kwargs):
         if data is MISSING or data is None:
             self._fail('required')
 
-        if isinstance(data, string_types):
+        if isinstance(data, string_types) or isinstance(data, bool):
```

Comment:
We can avoid an extra function call here by passing a tuple of tuples like the following:
```
if isinstance(data, (string_types, bool)):
```
Time measurements for Python 2.7.16:
```
>>> timeit.timeit('[x(i) for i in range(100)]', 's = (str,unicode); x = lambda x: isinstance(x, s) or isinstance(x, bool)')
42.79368710517883
>>> timeit.timeit('[x(i) for i in range(100)]', 's = (str,unicode); x = lambda x: isinstance(x, (s, bool))')
39.61967992782593
>>> timeit.timeit('[x(True) for i in range(100)]', 's = (str,unicode); x = lambda x: isinstance(x, s) or isinstance(x, bool)')
38.124322175979614
>>> timeit.timeit('[x(True) for i in range(100)]', 's = (str,unicode); x = lambda x: isinstance(x, (s, bool))')
34.10790705680847
```
For Python 3.7.3:
```
>>> timeit.timeit('[x(i) for i in range(100)]', 's = (str,); x = lambda x: isinstance(x, s) or isinstance(x, bool)')
23.825916450000022
>>> timeit.timeit('[x(i) for i in range(100)]', 's = (str,); x = lambda x: isinstance(x, (s, bool))')
21.865216470000007
>>> timeit.timeit('[x(True) for i in range(100)]', 's = (str,); x = lambda x: isinstance(x, s) or isinstance(x, bool)')
21.76908864500001
>>> timeit.timeit('[x(True) for i in range(100)]', 's = (str,); x = lambda x: isinstance(x, (s, bool))')
19.823041282999952
```